lua/smart-splits/config.lua

```diff
@@ -79,6 +79,16 @@ function M.set_default_multiplexer()
     return
   end
 
+  -- if running in a GUI instead of terminal TUI, disable mux
+  -- because you aren't in any terminal, you're in a Neovim GUI
+  local current_ui = vim.tbl_filter(function(ui)
+    return ui.chan == 1
+  end, vim.api.nvim_list_uis())[0]
```

Comment:
This should be `[1]` since lua is 1-indexed.
```suggestion
  end, vim.api.nvim_list_uis())[1]
```

Reply:
Whoops. That catches me off guard sometimes. So used to 0-based indexing 😛